backend_plugin/src/main/evaluators/BlockCommentEvaluator.java

```diff
@@ -29,72 +34,52 @@ public BlockCommentEvaluator() {
 	 * @return true if the user comments two sequential lines of code, false otherwise
 	 */
 	public boolean evaluate(DocumentEvent event) {
+		// Dev Notes: John
+		// If a user comments out a line with ctrl + /, then the length is 0 and the text is "//"
+		// A document event can't tell the difference between ctrl + / on two consecutive lines and
+		// commenting out a block of code all at once
+		// ModificationStamp is just a counter that increments. So that's not helpful
 
-		// Took logic Eric created to check for a double back slash
-		// May be a way to optimize this, but it works
-		int currentOffset = event.getOffset();
-		String currentInsert = event.getText();
-		if (prevOffset == currentOffset - 1) {
-			if (currentInsert != null && this.prevInsert != null &&
-					this.prevInsert.equals("/") && currentInsert.equals("/")) {
-
-				// DEBUG
-				System.out.println("Placed two slashes");
-
-				// Verify that the double backslash is at the start of the line
-				IDocument doc = event.getDocument();
-				try {
-					int lineOffset = doc.getLineInformationOfOffset(currentOffset).getOffset();
-					String textBeforeSlashes = doc.get(lineOffset, prevOffset-lineOffset);
-
-					// DEBUG
-					//					System.out.println("line offset: " + lineOffset);
-					//					System.out.println("Prev offset: " + prevOffset);
-					//					System.out.println("Text before slashes: " + textBeforeSlashes);
+		if (doubleBackSlash(event)) {
+			IDocument doc = event.getDocument();
+			try {
+				// Verify that the double slash is at the beginning of the line
+				int lineOffset = doc.getLineInformationOfOffset(currentOffset).getOffset();
```

Comment:
I would add a comment or change the variable name to make sure readers know that this gets offset of the start of the line